### DISCUSSION TITLE
Logs/Loki: Improve how results under "Inspect" are presented

### DISCUSSION THREAD

**Discussion by DylanGuedes** [grafana/grafana#66470] [Observability]
Hello,
I'd like to request an improvement to the "Inspect" results that are presented after running a Loki query. My motivations are:
- If you have multiple queries multiple results are presented. As of now, it is pretty hard to understand which query is the responsible for which result. A heading/separation reporting which query originated an specific result would be really helpful
- If you run a long query that is split into multiple subqueries, the subqueries results are presented merged. Ditto here: it is kinda hard to understand that the merged results originated from a single query and which time range created which statistics results.

If something sounds confusing please lmk so I can elaborate further.

Thank you

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82718. Feel free to continue the discussion around this there. Thank you!
